packages/studio-ui/src/components/UndoRedo.tsx

```diff
@@ -22,6 +22,20 @@ export default function UndoRedo(): JSX.Element {
     redo();
   }, [redo]);
 
+  const handleUndoKeydown = useCallback(
+    (event: KeyboardEvent) => {
+      if ((event.ctrlKey || event.metaKey) && event.key === "z") {
+        event.preventDefault();
+        undo();
+      }
+    },
+    [undo]
+  );
+
+  useEffect(() => {
+    document.addEventListener("keydown", handleUndoKeydown);
```

Comment:
can we return here so the `useEffect` cleans up the listener on unmount?

Reply:
Yes! I had it at one point, not sure why I removed it-- its back now, though!